exp-player/addon/helpers/breaklines.js

```diff
@@ -0,0 +1,10 @@
+// h/t http://stackoverflow.com/questions/12331077/does-handlebars-js-replace-newline-characters-with-br
```

Comment:
Have you confirmed that this works (in the context of a bound Ember attribute)? If so I will change it.